src/image/operator/cannyEdge.js

```diff
@@ -4,6 +4,9 @@ import cannyEdgeDetector from 'canny-edge-detector';
  * @memberof Image
  * @instance
  * @param {object} [options]
+ * @param {number} [options.gaussianBlur] - Sigma parameter for the gaussian filter step (default: 1.1)
```

Comment:
Default values should be defined like this:

`@param {number} [options.gaussianBlur=1.1] - ...`

Ditto for the other parameters.